backend/my-backend-api/lambda/createProduct.ts

```diff
@@ -0,0 +1,109 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import {PutCommand, TransactWriteCommand} from "@aws-sdk/lib-dynamodb";
+import { dynamoDB, PRODUCTS_TABLE, STOCKS_TABLE } from "./utils/dynamodbClient";
+
+interface Product {
+    id: string;
+    title: string;
+    description?: string;
+    price: number;
+}
+
+interface Stock {
+    product_id: string;
+    count: number;
+}
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+    // Handle CORS for OPTIONS request
+    if (event.httpMethod === "OPTIONS") {
+        return {
+            statusCode: 200,
+            headers: {
+                "Access-Control-Allow-Headers": "Content-Type",
+                "Access-Control-Allow-Origin": "*",
+                "Access-Control-Allow-Methods": "POST, OPTIONS",
+            },
+            body: "",
+        };
+    }
+
+    try {
+        // Parse the request body
+        const body = JSON.parse(event.body || "{}");
+
+        // Validate required fields
+        if (!body.id || !body.title || !body.price) {
```

Comment:
Usually guid id is not a part of the body in create request.  Better to be created here.

Reply:
Thank you so much! I've made changes
![image](https://github.com/user-attachments/assets/e74670fd-7da8-4aef-803e-a208903e3ea5)